helpers/installcredprovider.sh

```diff
@@ -16,8 +16,13 @@ NUGET_PLUGIN_DIR="$HOME/.nuget/plugins"
 
 # determine whether we install default or Net6
 if [[ ! -z ${USE_NET6_ARTIFACTS_CREDENTIAL_PROVIDER} ]]; then
-  FILE="Microsoft.Net6.NuGet.CredentialProvider.tar.gz"
-
+  if [[ -z "WINDIR" ]]
+  then
+    FILE="Microsoft.Net6.NuGet.CredentialProvider.tar.gz"
+  else
+    FILE="Microsoft.Net6-Windows.NuGet.CredentialProvider.tar.gz"
```

Comment:
@jmyersmsft It looks like `https://aka.ms/install-artifacts-credprovider.sh` redirects to `https://raw.githubusercontent.com/microsoft/artifacts-credprovider/master/helpers/installcredprovider.sh`.

This is bad because an official `Microsoft.Net6-Windows.NuGet.CredentialProvider.tar.gz` does not exist yet and so as soon as this merges, this will break:
```
REDMOND+jerick@jerick-hpz440 MINGW64 /d/src/artifacts-credprovider/helpers (broker)
$ USE_NET6_ARTIFACTS_CREDENTIAL_PROVIDER=1 ./installcredprovider.sh
Downloading from https://github.com/Microsoft/artifacts-credprovider/releases/latest/download/Microsoft.Net6-Windows.NuGet.CredentialProvider.tar.gz

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

I guess this installer changes need to come later?

Reply:
oh yeah, forgot about that

Reply:
ok commented out for now